tests/puppeteer/pages/BO/customers/addresses/add.js

```diff
@@ -0,0 +1,59 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddAddress extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitleCreate = 'Addresses •';
+    this.pageTitleEdit = 'Edit •';
+
+    // Selectors
+    this.customerEmailInput = '#customer_address_customer_email';
+    this.customerAddressdniInput = '#customer_address_dni';
+    this.customerAddressAliaInput = '#customer_address_alias';
```

Comment:
Did you mean to write 'Alias' ? if so a 's' is missing, it should be customerAddressAliasInput

Reply:
```suggestion
    this.customerAddressAliasInput = '#customer_address_alias';
```